sos/plugins/networking.py

```diff
@@ -17,10 +18,17 @@ class Networking(Plugin):
     plugin_name = "networking"
     profiles = ('network', 'hardware', 'system')
     trace_host = "www.example.com"
-    option_list = [(
+    option_list = [
         ("traceroute", "collect a traceroute to %s" % trace_host, "slow",
-         False)
-    )]
+         False),
+        ("namespace_regex", "Specific namespaces regex to be " +
+         "collected, namespaces regex should be separated by whitespace " +
+         "as for example \"eth* ens2\"", "fast", ""),
+        ("namespaces", "Number of namespaces to collect, 0 for unlimited. " +
+         "Incompatible with the namespaces plugin option", "slow", 0),
```

Comment:
Should read "incompatible with the namespaces_regex plugin option" following the option name changes.

Reply:
Updated in latest commit.